modules/v2x/v2x_proxy/proto_adapter/proto_adapter_test.cc

```diff
@@ -0,0 +1,64 @@
+/******************************************************************************
+ * Copyright 2020 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+/**
+ * @file proto_adapter_test.cc
+ */
+
+#include "modules/v2x/v2x_proxy/proto_adapter/proto_adapter.h"
+
+#include "gtest/gtest.h"
+
+namespace apollo {
+namespace v2x {
+using ::v2x::proxy::AdapterLightColorObu2os;
+using ::v2x::proxy::AdapterLightTypeObu2os;
+using ::v2x::proxy::OBULightColorE;
+using ::v2x::proxy::OSLightColorE;
+using ::v2x::proxy::OSLightTypeE;
+
+TEST(v2x_proxy_app, v2x_proxy_adapter_convert_enum) {
+  EXPECT_EQ(
+      OSLightColorE::SingleTrafficLight_Color_RED,
+      AdapterLightColorObu2os(OBULightColorE::SingleTrafficLight_Color_RED));
+  EXPECT_EQ(OSLightColorE::SingleTrafficLight_Color_YELLOW,
+            AdapterLightColorObu2os(
+                OBULightColorE::SingleTrafficLight_Color_YELLOW));
+  EXPECT_EQ(OSLightColorE::SingleTrafficLight_Color_GREEN,
+            AdapterLightColorObu2os(
+                OBULightColorE::SingleTrafficLight_Color_GREEN));
+  EXPECT_EQ(OSLightColorE::SingleTrafficLight_Color_BLACK,
+            AdapterLightColorObu2os(
+                OBULightColorE::SingleTrafficLight_Color_BLACK));
+  EXPECT_EQ(OSLightColorE::SingleTrafficLight_Color_FLASH_GREEN,
+            AdapterLightColorObu2os(
+                OBULightColorE::SingleTrafficLight_Color_FLASH_GREEN));
+  EXPECT_EQ(OSLightColorE::SingleTrafficLight_Color_UNKNOWN,
+            AdapterLightColorObu2os(
+                OBULightColorE::SingleTrafficLight_Color_UNKNOWN));
+  ////////////////////////////////////////////////////////////////
+  EXPECT_EQ(OSLightTypeE::SingleTrafficLight_Type_LEFT,
+            AdapterLightTypeObu2os(2));
+  EXPECT_EQ(OSLightTypeE::SingleTrafficLight_Type_RIGHT,
+            AdapterLightTypeObu2os(3));
+  EXPECT_EQ(OSLightTypeE::SingleTrafficLight_Type_U_TURN,
+            AdapterLightTypeObu2os(4));
+  EXPECT_EQ(OSLightTypeE::SingleTrafficLight_Type_STRAIGHT,
+            AdapterLightTypeObu2os(1));
```

Comment:
If  `os` in `2os` means operating system, please rewrite it as `Obu2OS`. or `Obu2Sys`. It's hard to differentiate it with `20s, 2os, 2Os.`